### PR TITLE
Updates button examples and changes file to .md

### PR DESCRIPTION
**Pull request by alexandervalencia**

### CODE REVIEW

**Comment by Robert-Frampton** [comment]
@pat270 are buttons ready to be migrated?

**Comment by alexandervalencia** [comment]
Thanks for double-checking on this.

FYI @Robert-Frampton We've been using a [google sheet](https://docs.google.com/spreadsheets/d/1-fWylAjIyyrboj7wOvAh0M4owJg1nHeXLng0hY394gI/edit#gid=0) to keep track of what's ready. @pat270 marks the "Clay-Ready" column when we can start working on them.

File: src/pages/docs/components/buttons.md [hunk 1]
```diff
@@ -0,0 +1,990 @@
...
+	<button class="btn btn-outline-warning" disabled type="button">Warning</button>
+	<button class="btn btn-outline-danger" disabled type="button">Danger</button>
+	<div class="clay-site-light-container">
+		<button class="btn btn-outline-light" disabled type="button">Light</button>
+	</div>
+	<button class="btn btn-outline-dark" disabled type="button">Dark</button>
+</div>
+
+```xml
+<div style="margin-bottom:5px;">
```
Comment (Robert-Frampton) [inline src/pages/docs/components/buttons.md]: We should get rid of these divs from the code examples.

Comment (Robert-Frampton) [inline src/pages/docs/components/buttons.md]: In fact, can we just get rid of them from the source as well?

**Comment by alexandervalencia** [comment]
Hey @Robert-Frampton,

I just pushed up the change that removes all the unnecessary divs 👍

**Pull request merged**